pkgs/by-name/na/nautilus-open-in-blackbox/package.nix

```
@@ -0,0 +1,33 @@
+{ python3, fetchFromGitHub, gnome, stdenv, lib }:
+stdenv.mkDerivation rec {
+  pname = "nautilus-open-in-blackbox";
+  version = "0.1.1";
+
+  src = fetchFromGitHub
+    {
+      owner = "ppvan";
+      repo = pname;
```

```suggestion
      repo = "nautilus-open-in-blackbox";
```
as repo and pname are semantically different